tests/integration/management/commands/test_run_file_uploader.py

```diff
@@ -77,15 +77,17 @@ def test_do_upload_task(upload_files_stubber, bll, freezer):
 
 
 def test_do_upload_task_updated_file_content(upload_files_stubber, bll):
-    release_request, workspace, upload_files_responses = setup_release_request(
+    release_request, upload_files_responses = setup_release_request(
         upload_files_stubber, bll, response_statuses=[201]
     )
 
     relpath = UrlPath("test/file.txt")
     # modify workspace file content
-    factories.write_workspace_file(release_request.workspace, relpath, contents="changed")
+    factories.write_workspace_file(
+        release_request.workspace, relpath, contents="changed"
+    )
     # refresh workspace
-    workspace = bll.get_workspace(
+    bll.get_workspace(
```

Comment:
can we drop this `get_workspace` call entirely?

Reply:
We could, but I left it in there after the failing test was passing just in case. I have another PR about to go, so I might remove it there, but leaving it here to avoid the re-approve faff